modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/schema/TableDescriptorImpl.java

```diff
@@ -127,7 +135,7 @@ public RelDataType deleteRowType(IgniteTypeFactory factory) {
     /** {@inheritDoc} */
     @Override
     public IgniteDistribution distribution() {
-        return IgniteDistributions.random();
+        return IgniteDistributions.affinity(colocationColumns, 0, dummyAffinityIdentity);
```

Comment:
why we use 0 here?

Reply:
I've added comment explaining such choice. Also check please comment in beginning of the class